app/Repositories/Activity/ActivityRepository.php

```diff
@@ -355,6 +357,50 @@ public function populateOrderNumber()
         }
     }
 
+    /**
+     * Gets activities for LTI external tool search from CurrikiGo
+     * @param array $data
+     * @return array
+     */
+    public function ltiSearchForm ($request) {
+        // Fetch Elastic Search results
+        $data = [
+            'query' => $request->input('query', ''),
+            'from' => $request->input('from', 0),
+            'size' => 10,
+            'model' => 'activities',
+            'indexing' => intval($request->input('private', 0)) === 1 ? [] : [3]
+        ];
+
+        // Check LMS settings for authorization when searching private projects
+        if (empty($data['indexing'])) {
+            $lmsSetting = LmsSetting::where('lti_client_id', $request->input('ltiClientId'))->first();
+
+            if (empty($lmsSetting))
+                $data['indexing'] = [3]; // Switching to public only
+            else
+                $data['userIds'] = [$lmsSetting->user_id];
+        }
+
+        // If a an author is provided, limit to projects from that user only
+        if ($request->has('author')) {
+            $author = User::where('email', $request->input('author'))->first();
+            if(!empty($author))
+                $data['userIds'] = [$author->id];
+        }
+
+        $data['subjectIds'] = $request->has('subject') ? [$request->input('subject')] : [];
+        $data['educationLevelIds'] = $request->has('level') ? [$request->input('level')] : [];
+
+        if ($request->has('start'))
+            $data['startDate'] = $request->input('start', '');
+
+        if ($request->has('end'))
+            $data['endDate'] = $request->input('end', '');
+
```

Comment:
Space needed after if.
And opening/closing blocks should be added for if/else statements.